#FCK in velocity tools configurable

=svnm= https://source.sakaiproject.org/svn/reference/trunk/@45587 reference

=svnm= https://source.sakaiproject.org/svn/velocity/trunk/@45588 velocity

#Fill in the SAK or something unique, leave it like this if theres no real patch
--- /dev/null
+++ SAK-13244.patch 
@@ -0,0 +1 @@
+ 
